mozilla_django_oidc_db/views.py

```diff
@@ -204,35 +202,35 @@ def my_digid_login(request):
 
     .. code-block:: python
 
-        digid_init = OIDCInit.as_view(
-            config_class=OpenIDConnectPublicConfig, allow_next_from_query=False
+        digid_init = OIDCAuthenticationRequestInitView.as_view(
+            identifier="config-identifier", allow_next_from_query=False
         )
 
         def my_digid_login(request):
             return digid_init(request, return_url="/some-fixed-url")
     """
 
-    def get_settings(self, attr: str, *args: Any) -> Any:  # type: ignore
-        """
-        Look up the request setting from the database config.
+    oidc_rp_scopes: str = None
```

Comment:
`None` is not assignable to `str`, so this type hint is a lie

```suggestion
    oidc_rp_scopes: str = ""
```